cmd_createitem.go

```diff
@@ -21,15 +20,19 @@ func addItemsToNetlist(c *cli.Context) error {
 
 	common.VerifyArgumentByName(c, "id")
 
-	if c.String("from-file") == "" {
+	if len(c.StringSlice("from-file")) < 1 {
 		common.VerifyArgumentByName(c, "items")
 
 		if len(c.StringSlice("items")) < 1 {
 			log.Fatal("Please provide items!")
 
 		}
+
+		//Add out items to slice
+		itemsToAdd = strings.Split(c.StringSlice("items")[0], ",")
+
 	} else {
-		// This is the way we access first of the params in CLI
+		//TODO: This is the way we access first of the params in CLI - check if we can do it cleaner
 		path := c.StringSlice("from-file")[0]
```

Comment:
Why not to use just string for from-file? Do we plan to have several files as source?

Reply:
Changed to proper string flag value